tools/ci-build/scripts/generate-aws-sdk

```diff
@@ -27,12 +27,17 @@ mv aws-doc-sdk-examples/rust_dev_preview smithy-rs/aws/sdk/examples
 rm -rf smithy-rs/aws/sdk/examples/.cargo
 rm smithy-rs/aws/sdk/examples/Cargo.toml
 
+echo -e "${C_YELLOW}Creating empty model metadata file since we don't have model update information...${C_RESET}"
+MODEL_METADATA_PATH="$(pwd)/model-metadata.toml"
```

Comment:
The model metadata is produced by the automation that brings in the latest SDK service models, and includes information such as what kind of change it was (documentation only, or APIs), and a changelog entry describing the change.

The `generate-aws-sdk` script is only used in CI, and validates that a SDK can be successfully generated. It's not used to generate the SDK for releases since the `sdk-sync` tool does that, and there, model metadata and codegen changes get incorporated.